op-batcher/readme.md

```diff
@@ -0,0 +1,40 @@
+# op-batcher
```

Comment:
Thank you so much for adding this and the architecture diagram @geoknee - super helpful! 🙌🏼 🎉